catalog/imaging/imaging-plugin-nitf/src/main/java/org/codice/alliance/plugin/nitf/NitfPostIngestPlugin.java

```diff
@@ -285,6 +285,14 @@ private BufferedImage renderImageUsingOriginalDataModel(InputStream source)
             return input.getRight().renderToClosestDataModel(input.getLeft());
           } catch (IOException e) {
             LOGGER.debug(e.getMessage(), e);
+          } catch (Error e) {
```

Comment:
❗️Could we catch `RuntimeException` and specific exceptions here instead of `Error`? If not, we should catch and re-throw `VirtualMachineErrors` and catch anything else in a separate `catch (Throwable t)` block, i.e.,

```
try {
  // ...
} catch (VirtualMachineError e) {
  throw e;
} catch (Throwable t} {
  // Log...
}
```

Reply:
The library caches IOExceptions and then throws a RuntimeException. It then catches RuntimeExceptions and throws an Error so it will never throw a RuntimeException. I will update as suggested.